workshops/views.py

```diff
@@ -1061,16 +1062,19 @@ def event_import(request):
 
         if 'slug' not in tags:
             # there are no HTML tags, so let's try the old method
-            url = generate_url_to_event_index(url)
+            index_url = generate_url_to_event_index(url)
 
             # fetch page
-            response = requests.get(url)
+            response = requests.get(index_url)
 
             if response.status_code == 200:
                 # don't throw errors for pages we fall back to
                 content = response.text
                 tags = find_tags_on_event_index(content)
 
+                if 'slug' not in tags:
+                    tags['slug'] = urlparse(url).path.replace('/', '')
+
         # normalize (parse) them
         tags = parse_tags_from_event_website(tags)
 
```

Comment:
Also tangential to this PR, but we probably want to check for and handle the case where we get up to `parse_tags_from_event_website` but don't actually have a tag dictionary yet (e.g. we got a non-200 status code for the old method).